packages/stargate/src/stargateclient.ts

```diff
@@ -49,8 +49,14 @@ function coinFromProto(input: cosmos.ICoin): Coin {
   };
 }
 
+/** Use for testing only */
+export interface PrivateStargateClient {
```

Comment:
If this is only for testing, why not put it in a test file?

Reply:
Last time we did this (`export interface PrivateCosmWasmClient {`), it was needed in multiple test files. Also I think it is nice to have it close to the definitions of the fields. In this case we could move it, but if you don't have a strong opinion here, I'd leave it.